### DISCUSSION TITLE
html-portfolio

### DISCUSSION THREAD

**Discussion by amarhussain01** [community/community#56281] [Programming Help]
### Hi everyone,

I have a site called html-portfolio. I am having issues with the home link and contact link. These are html pages aswell as index.html. they are not opening whilst been published on github. Reading 404 on pages
Also the pictures i have linked and not displaying on index.html

**Comment by radar-blip** [comment]
Some of your links (in index.html) have a backslash instead of a forward slash.
Try changing this:
`<img src="../publish\cat.png" width="200" height="200" alt="cat picture">`

to this:
`<img src="./publish/cat.png" width="200" height="200" alt="cat picture">`


Also, change this:
`<li><a href="C:\Users\amarh\OneDrive\Desktop\angel yu course\publish\contact.html">Contact</a></li>`

to this:
`<li><a href="contact.html">Contact</a></li>`

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @amarhussain01 , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
